fdsys/examples/2006/ca8-03-03578.xml

```diff
@@ -0,0 +1,165 @@
+<mods xmlns="http://www.loc.gov/mods/v3" xmlns:exslt="http://exslt.org/common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-3.xsd" version="3.3" ID="P0b002ee186bff94d">
```

Comment:
Can you please put all these example files into a separate PR? I can't review this one because there's too much content here.